src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -103,4 +118,16 @@ private static function isExpired($date)
 
         return false !== $date && new \DateTime() > $date->modify('last day of this month 23:59:59');
     }
+
+    private static function getDotEnvVars()
+    {
+        $vars = array();
+        foreach (explode(',', getenv('SYMFONY_DOTENV_VARS')) as $name) {
```

Comment:
this will break when `getenv` returns `false`

Reply:
@stof, explode returns an array with one empty string:
```
>>> explode(',', false)
=> [
     "",
   ]
>>> 
```